apis/user-java/src/main/resources/application.properties

```diff
@@ -5,6 +5,6 @@ spring.jackson.date-format=io.swagger.RFC3339DateFormat
 spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 spring.datasource.driver-class-name=com.microsoft.sqlserver.jdbc.SQLServerDriver
 spring.jpa.database-platform=org.hibernate.dialect.SQLServer2012Dialect
-spring.datasource.url=someurl
-spring.datasource.username=username
-spring.datasource.password=password
+spring.datasource.url=${SQL_SERVER}
```

Comment:
This is going to fail since the value doesn't contain the full string it needs as you had earlier.